### ISSUE TITLE
Kendo Drawer -> items null -> exception

### ISSUE THREAD

**Issue by meriturva** [telerik/kendo-angular#2727]
Just see https://stackblitz.com/edit/angular-og6c8j

```
 <kendo-drawer #drawer
            [items]="test?.items"
            [mode]="'push'"
            [mini]="true"
            [expanded]="true"
            (select)="onSelect($event)">
        </kendo-drawer>
```

When we try to pass 'null' as items input on kendo drawer component we get always an exception (ERROR TypeError: Cannot read property 'length' of null).

The only workaround now is to put a ternary if: `test != null ? test.items : ''`

**Comment by Raisolution** [comment]
Thank you for reporting this. Issue is fixed in `v4.2.1`.

**Comment by siyka-marinova** [comment]
Fixed in `kendo-angular-layout v4.2.1`
